src/v2/view-builder/views/password/EnrollAuthenticatorPasswordView.js

```diff
@@ -11,6 +12,37 @@ const Body = BaseForm.extend({
     return loc('oie.next.button', 'login');
   },
 
+  initialize () {
+    BaseForm.prototype.initialize.apply(this, arguments);
+    const policy = this.getPasswordPolicy();
+    this.displayPasswordPolicy(policy);
+  },
+
+  displayPasswordPolicy (policy) {
+    if (policy) {
+      let listHtml = '';
+      const rulesList = getPasswordComplexityDescriptionForHtmlList( policy );
+      rulesList.forEach(rule => listHtml += `<li>${rule}</li>`);
+  
+      this.add(
+        `<section class="password-authenticator__rules">
+          <div class="password-authenticator__rules-heading">
+            ${loc('password.complexity.requirements.header', 'login')}
+          </div>
+          <ul class="password-authenticator__rules-list">${listHtml}</ul>
+        </section>`,
+        {
+          prepend: true,
+          selector: '.o-form-fieldset-container',
```

Comment:
Instead of prepend, can we add in a container and then add this list to the container?

Reply:
I'm not sure I understand, can you give me an example pls? I don't know what value it will add though.. I'm just using what courage offers.

Reply:
@santhoshbalakrishnan this entire section for password requirement already inside a container `<section class="password-authenticator__rules">`

iirc, in v1, this list was added to `subtitle` container. does it work here?

Reply:
@anipendakur-okta Here's an example:

```
template: '<div class="callout-container"></div>'
...
this.add(callout, '.callout-container');
```

This way, the callout renders within the callout container.

@haishengwu-okta Unfortunately, subtitle might not work here since, we subtitles only take in a string and not a view. So we had to explicitly add it as a child.

Reply:
where will `<div class="callout-container"></div>` be added ?

the end result looks similar
either
```
<div class="callout-container">
 ...// policy list
</div>
```

or 

```
<section class="password-authenticator__rules">
... // policy list
</section>
```

And such entire container will be added to somewhere in the form.

Reply:
Ah cool, misunderstood earlier. Looks good.